main.tf

```diff
@@ -23,6 +23,7 @@ locals {
   ml_secrets_arn                   = data.terraform_remote_state.common.outputs.ml_secrets_arn
   bluenaas_service_secrets_arn     = data.terraform_remote_state.common.outputs.bluenaas_service_secrets_arn
   accounting_service_secrets_arn   = data.terraform_remote_state.common.outputs.accounting_service_secrets_arn
+  entitycore_service_secrets_arn   = data.terraform_remote_state.common.outputs.entitycore_service_secrets_arn
```

Comment:
For this to work we need to create the entitycore_service_secrets_arn in the deployment-common. I can take care of it, just let me know :)

Reply:
done here https://github.com/openbraininstitute/aws-terraform-deployment-common/pull/41